### PR TITLE
WRN-1353: Migrated tests for ProgressButton and ProgressBar.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1353

### CODE REVIEW

File: ProgressButton/tests/ProgressButton-specs.js [hunk 1]
```diff
@@ -1,77 +1,85 @@
...
-		const progressButton = shallow(
+		const {getByRole} = render(
 			<ProgressButtonBase showProgress progress={0.5}>
 				Progress Button
 			</ProgressButtonBase>
 		);
 
-		const actual = progressButton.find('.icon').prop('children');
-		expect(actual).toBe(expected);
+		const expectedCode = 983004;
```
Comment (adrian-cocoara-lgp) [inline ProgressButton/tests/ProgressButton-specs.js:66]: Add a comment here: // // decimal converted charCode of Unicode 'stop' character

File: ProgressButton/tests/ProgressButton-specs.js [hunk 2]
```diff
@@ -1,77 +1,85 @@
...
+		const customIcon = 'star';
+		const {getByRole} = render(
+			<ProgressButtonBase showProgress progress={0.5} icon={customIcon}>
 				Progress Button
 			</ProgressButtonBase>
 		);
 
-		const actual = progressButton.find('.icon').prop('children');
-		expect(actual).toBe(expected);
+		const expectedCode = 983080;
```
Comment (adrian-cocoara-lgp) [inline ProgressButton/tests/ProgressButton-specs.js:80]: Add a comment gere: // decimal converted charCode of Unicode 'play' character

Comment (alexandrumorariu) [reply ProgressButton/tests/ProgressButton-specs.js:82]: Done

File: ProgressButton/tests/ProgressButton-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply ProgressButton/tests/ProgressButton-specs.js:67]: Done

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**